packages/plugin-image-generation/src/index.ts

```diff
@@ -116,9 +126,15 @@ const imageGeneration: Action = {
         const images = await generateImage(
             {
                 prompt: imagePrompt,
-                width: 1024,
-                height: 1024,
-                count: 1,
+                ...(options.width !== undefined ? { width: options.width || 1024 } : {}),
```

Comment:
instead of !== undefined you probably want to do != null which checks for undefined or null